src/Mod/Sketcher/Gui/Command.cpp

```diff
@@ -971,6 +971,7 @@ class GridSpaceAction : public QWidgetAction
 {
 public:
     GridSpaceAction(QObject* parent) : QWidgetAction(parent) {
+        setEnabled(false);

```

Comment:
@0penBrain 
This is the alternative to the eventfiltering. Disabling the action itself.

Reply:
Great, much much better.
Did you find this in some doc? 
Still not clear for me the mechanism that make it call the "parent" action. Tjx

Reply:
If enabled, the QAction will upon LMB trigger. When triggered there is no stop (unless you go the eventfiltering route), it triggers the actiongroup, which triggers the command.

I wanted to avoid the eventfilter, so I searched for disabling/deactivating a QAction.

Reply:
Thanks for details. I can only agree to avoid using event filters. ;)